### PR TITLE
OCLOMRS-301: Enable saving of different languages for concept name

### PR DESCRIPTION
**Pull request by dessHub**
OCLOMRS-301: Enable saving of different languages for concept name

# JIRA TICKET NAME:
[OCLOMRS-301: Enable saving of different languages for concept name](https://issues.openmrs.org/browse/OCLOMRS-301)

# Summary:
- Fix concept locale being displayed on edit form.

### CODE REVIEW

**Comment by abulojoshua1** [comment]
@dessHub, kindly attach screenshots or video showing the before_fix and after_fix of your changes to [this JIRA ticket](https://issues.openmrs.org/browse/OCLOMRS-301). You can do so by Uploading the `drop file to attach, or browse` button.

**Comment by dessHub** [comment]
> @dessHub, kindly attach screenshots or video showing the before_fix and after_fix of your changes to [this JIRA ticket](https://issues.openmrs.org/browse/OCLOMRS-301). You can do so by Uploading the `drop file to attach, or browse` button.

**Comment by dessHub** [comment]
> @dessHub, kindly attach screenshots or video showing the before_fix and after_fix of your changes to [this JIRA ticket](https://issues.openmrs.org/browse/OCLOMRS-301). You can do so by Uploading the `drop file to attach, or browse` button.

Done

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,10 @@ package-lock.json
 **/__snapshots__/**
 src/components/dashboard/container/ListContainer.jsx
 .env
+
+tags
+tags.lock
+tags.temp
+
+*.swp
+*.swo
```
Comment (muhozi) [inline .gitignore]: Is this related to the ticket?

Comment (dessHub) [reply .gitignore]: no. Those are vim generated files so I didn't want them to end up on the repo.

Comment (dessHub) [reply .gitignore]: Hey @muhozi, I have fixed it. I created a global `.gitignore` file for my editor's generated files.
